test/integration/trigger.test.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+var chai = require('chai')
+  , Sequelize = require('../../index')
+  , expect = chai.expect
+  , Support = require(__dirname + '/../support')
+  , dialect = Support.getTestDialect();
+
+describe(Support.getTestDialectTeaser('Model'), function() {
```

Comment:
You should wrap this whole test file in `if dialect.__supports.tmpTableTrigger` so the test are only run for mssql